packages/components/.storybook/main.js

```diff
@@ -3,11 +3,27 @@ const path = require("path");
 module.exports = {
   stories: ["../src/*.stories.mdx", "../src/**/*.stories.(ts|tsx|mdx)"],
   addons: ["@storybook/preset-typescript", "@storybook/addon-docs"],
-  webpackFinal: config => {
+  webpackFinal: (config) => {
     config.module.rules.push({
       test: /\.scss$/,
-      use: ["style-loader", "css-loader", "sass-loader"],
       include: path.resolve(__dirname, "../"),
+      use: ["style-loader", "css-loader", "sass-loader"],
+    });
+
+    config.module.rules.push({
```

Comment:
Follows the [Linaria guide for webpack setup](https://github.com/callstack/linaria/blob/master/docs/BUNDLERS_INTEGRATION.md#webpack)

Something isn't compiling correctly though :( I get a bunch of TS errors from `button.tsx` that don't make any sense